### ISSUE TITLE
Why use the  train_Imagenet.py can not achieve high top-5/top-1 accuracy？

### ISSUE THREAD

**Issue by johnjunjun7** [david8862/keras-YOLOv3-model-set#26]
I try to use the train_Imagenet to train the shufflenetv2 network. The batch size is 128, and other parameters are default. The top-5 of the verification set is only 74.7%, the top-1 is only 50%, which is far from 70% of the top-1 of the paper. Is the batch size used in the training too small ?

**Comment by david8862** [comment]
> I try to use the train_Imagenet to train the shufflenetv2 network. The batch size is 128, and other parameters are default. The top-5 of the verification set is only 74.7%, the top-1 is only 50%, which is far from 70% of the top-1 of the paper. Is the batch size used in the training too small ?

@johnjunjun7, good question. Actually I didn't deeply dig into this topic. But generally I guess some configs/options should also impact the final Imagenet accuracy, including optimizer type, learning rate decay or some other training strategy/tricks. Batch size is also one important factor.

**Comment by johnjunjun7** [comment]
If the backbone training is not good, will it affect the effect of the later detection network?

**Comment by david8862** [comment]
> If the backbone training is not good, will it affect the effect of the later detection network?

From the experience of [yolo3_nano](https://github.com/david8862/keras-YOLOv3-model-set/releases/download/v1.0.1/yolo3_nano_weights_416_voc.tar.gz) and some other models (I've ever did similar thing on [TF detection model zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md)), that should not be a big concern. From my understanding, the imagenet pretraining just provides a common feature representation on backbone, and still need to aligned with the detection dataset during further training.
